articles/partner-twilio-java-how-to-use-voice-sms.md

```diff
@@ -75,101 +76,88 @@ When you sign up for a Twilio account, you'll receive an account ID and an authe
+    String body = "Where's Wallace?";
 
-    List<NameValuePair> params = new ArrayList<NameValuePair>();
-    params.add(new BasicNameValuePair("To", "+14159352345")); // Replace with a valid phone number for your account.
-    params.add(new BasicNameValuePair("From", "+14158141829")); // Replace with a valid phone number for your account.
-    params.add(new BasicNameValuePair("Body", "Where's Wallace?"));
+    // Create a Message creator passing From, To and Body values
+    // then send the SMS message by calling the create() method
+    Message sms = Message.creator(to, from, body).create();
 
-    Message sms = messageFactory.create(params);
-
-For more information about the parameters passed in to the **SmsFactory.create** method, see [http://www.twilio.com/docs/api/rest/sending-sms][twilio_rest_sending_sms].
+For more information about the parameters passed in to the **Message.creator** method, see [http://www.twilio.com/docs/api/rest/sending-sms][twilio_rest_sending_sms].
 
 ## <a id="howto_provide_twiml_responses"></a>How to: Provide TwiML Responses from your own Website
-When your application initiates a call to the Twilio API, for example via the **CallFactory.create** method, Twilio will send your request to a URL that is expected to return a TwiML response. The example above uses the Twilio-provided URL [http://twimlets.com/message][twimlet_message_url]. (While TwiML is designed for use by Web services, you can view the TwiML in your browser. For example, click [http://twimlets.com/message][twimlet_message_url] to see an empty **&lt;Response&gt;** element; as another example, click [http://twimlets.com/message?Message%5B0%5D=Hello%20World][twimlet_message_url_hello_world] to see a **&lt;Response&gt;** element that contains a **&lt;Say&gt;** element.)
+When your application initiates a call to the Twilio API, for example via the **CallCreator.create** method, Twilio will send your request to a URL that is expected to return a TwiML response. The example above uses the Twilio-provided URL [http://twimlets.com/message][twimlet_message_url]. (While TwiML is designed for use by Web services, you can view the TwiML in your browser. For example, click [http://twimlets.com/message][twimlet_message_url] to see an empty **&lt;Response&gt;** element; as another example, click [http://twimlets.com/message?Message%5B0%5D=Hello%20World][twimlet_message_url_hello_world] to see a **&lt;Response&gt;** element that contains a **&lt;Say&gt;** element.)
 
 Instead of relying on the Twilio-provided URL, you can create your own URL site that returns HTTP responses. You can create the site in any language that returns HTTP responses; this topic assumes you'll be hosting the URL in a JSP page.
 
 The following JSP page results in a TwiML response that says **Hello World** on the call.
 
     <%@ page contentType="text/xml" %>
-    <Response> 
+    <Response>
         <Say>Hello World</Say>
```

Comment:
perhaps this `Hello World` could be changed to `Hello World!` here and in other parts of the tutorial?